lib/generated/intl/messages_en.dart

```diff
@@ -74,6 +74,12 @@ class MessageLookup extends MessageLookupByLibrary {
         "about": MessageLookupByLibrary.simpleMessage("About"),
         "account_suspended":
             MessageLookupByLibrary.simpleMessage("Account suspended"),
+        "activate_non_confirmation_bias_mode_description":
+            MessageLookupByLibrary.simpleMessage(
+                "Don\'t show the author of the tweet on a post. Avoid confirmation bias based on authoritative arguments."),
+        "activate_non_confirmation_bias_mode_label":
+            MessageLookupByLibrary.simpleMessage(
+                "Activate non-confirmation-bias-mode"),
```

Comment:
I think this should just be `Activate non-confirmation bias mode`, as this is an open compound